### PR TITLE
Add staart

### PR DESCRIPTION
**Pull request by nmaro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@
 * [Next Express Bootstrap Boilerplate](https://github.com/MustansirZia/next-express-bootstrap-boilerplate) - Boilerplate for a full stack app built using Next, Express, react-bootstrap, SCSS and SSR with eslint.
 * [Next Blog Firestore](https://github.com/suevalov/next-blog-firestore) - Blog with simple CMS built with Next.js, Firebase Firestore, styled-components and mobx-state-tree.
 * [Next Redux Starter](https://github.com/CodementorIO/nextjs-redux-starter) - Next.js starter with Express, Redux, and PostCSS.
+* [Staart](https://github.com/nmaro/staart) - 😎 Actively maintained Next.js components library *and* minimal boilerplate to rapidly get staarted with app with working user accounts based on Ooth.
```
Comment (unicodeveloper) [inline README.md]: Based on Ooth? What's Ooth?

**Comment by unicodeveloper** [comment]
Also, can you show me where Next.js was used in the project? Went through the codebase and I didn't notice any use of Next.js anywhere.

**Comment by nmaro** [comment]
To answer your questions precisely:

1.

[Ooth](https://github.com/nmaro/ooth) is a user authentication / user accounts framework built on top of passport.js. It has itself next specific packages like [ooth-client-react-next-apollo](https://github.com/nmaro/ooth/tree/master/packages/ooth-client-react-next-apollo) and [ooth-client-react-next](https://github.com/nmaro/ooth/tree/master/packages/ooth-client-react-next).

2.

The point of staart is to be a *minimal* boilerplate that delegates as much as possible to libraries that can be maintained. So in the boilerplate itself you will find:

* the next.js setup: https://github.com/nmaro/staart/blob/master/examples/staart/index.js
* next router provider: https://github.com/nmaro/staart/blob/master/examples/staart/providers/next.js
* user authentication logic provider *for next*: https://github.com/nmaro/staart/blob/master/examples/staart/providers/ooth.js
* apollo provider *for next*: https://github.com/nmaro/staart/blob/master/examples/staart/providers/apollo.js

In the library you will find components that in principle depend on next.js (but the dependency is decoupled and handled through the context/providers defined in the boilerplate), e.g. the [with-login-required](https://github.com/nmaro/staart/blob/master/packages/staart/src/hocs/login-required.js) hoc depends on a user object that is injected in the context by the aforementioned ooth provider.

**Comment by unicodeveloper** [comment]
Oh, my bad! Thanks for the concise explanation @nmaro

**Comment by nmaro** [comment]
Thank you!

**Pull request merged**